runtime/commonMain/src/kotlinx/serialization/json/Json.kt

```diff
@@ -276,3 +210,47 @@ private val defaultJsonModule = serializersModuleOf(
 )
 
 internal const val lenientHint = "Use 'JsonConfiguration.isLenient = true' to accept non-compliant JSON"
+
+
+internal class JsonImpl(
```

Comment:
imo, `JsonImpl` in stacktraces looks very enterprise-ish and not nice to users, but I don't know how can we solve this.